flake.nix

```diff
@@ -324,6 +324,7 @@
             ++ pkgs.nixComponents.nix-external-api-docs.nativeBuildInputs
             ++ [
               pkgs.buildPackages.cmake
+              pkgs.shellcheck
```

Comment:
I wanted to run shellcheck myself manually.

Shellcheck has a diff output style that you can try to apply for auto fixes for lots of cases now
```shellcheck misc/systemv/nix-daemon -f diff | git apply```